frontend/src/components/dashboard/ActivityRow.vue

```diff
@@ -0,0 +1,127 @@
+<template>
+  <tr class="row">
+    <th style="text-align: left" class="tabular-nums" scope="row">
+      <span class="baseline-center"
+        ><span class="smaller">{{ scheduleEntry.number }}</span></span
+      >
+      <br />
+      <CategoryChip small dense :category="category" class="d-sm-none" />
+    </th>
+    <td class="d-none d-sm-table-cell">
+      <CategoryChip small dense :category="category" />
+    </td>
+    <td class="nowrap">
+      {{ start }}<br />
+      <span class="e-subtitle">{{ duration }}</span>
+    </td>
+    <td style="width: 100%" class="contentrow">
+      <router-link :to="routerLink" class="text-decoration-none black--text">
+        {{ title }}<br />
+      </router-link>
+      <span class="e-subtitle">{{ location }}</span>
+    </td>
+    <td class="contentrow avatarrow overflow-visible">
+      <AvatarRow :camp-collaborations="collaborators" size="28" class="ml-auto" />
+    </td>
+  </tr>
+</template>
+
+<script>
+import AvatarRow from './AvatarRow.vue'
```

Comment:
I do so more or less at random. I try to avoid too many ../../../../, and especially try to import from the symlinked common instead of the common folder outside the frontend folder. But most of the time I let phpstorm manage this

Reply:
Just my 2 cents:
- Relative paths make refactoring easy, if the components are tightly coupled (mostly for components in the same folder or subfolders). In this example I could rename or move the dashboard folder, and things just work (1 path only to fix).
- For anything else (usage of generic components, base components, etc.) I normally use @ because it's easier to search & replace in case of refactoring. Any ../../../../ thing is super difficult to catch and relies a lot on IDE magic for refactoring.